BackEndApp/authv1/views.py

```diff
@@ -71,9 +71,13 @@ def register(request):
         message = "Some error occurred!! Please try again."
         status = 401
         token = None
-    return JsonResponse(
-        {"message": message, "username": username, "token": token}, status=status
-    )
+        return JsonResponse(
```

Comment:
Wouldn't it be better to just set username = None in the except block? Rest everything can remain same

Reply:
Yeah I think that might be better. Also do we actually need it? Because it's already included in the request JSON

Reply:
Umm, you're right but I'd say let's not change the response altogether, might break something else in the front end 😅